servers/display_server.cpp

```diff
@@ -1334,6 +1334,24 @@ bool DisplayServer::can_create_rendering_device() {
 	}
 
 	Error err;
+
+#ifdef WINDOWS_ENABLED
+	// On some NVIDIA drivers combining OpenGL and RenderingDevice can result in crash, offload the check to the subprocess.
+	List<String> arguments;
+	arguments.push_back("--test-rd-creation");
+
+	String pipe;
+	int exitcode = 0;
+	err = OS::get_singleton()->execute(OS::get_singleton()->get_executable_path(), arguments, &pipe, &exitcode);
+	if (err == OK && exitcode == 0) {
+		created_rendering_device = RenderingDeviceCreationStatus::SUCCESS;
+		return true;
+	} else {
+		created_rendering_device = RenderingDeviceCreationStatus::FAILURE;
+		return false;
+	}
+#endif
```

Comment:
I'd suggest making it a static bool so we only test this once per session, to avoid redoing such test all the time.

Actually, the whole function should likely be called only once and saved in a static bool. It's called in `LightmapGI::get_configuration_warnings` so any time a scene with a LightmapGI is refreshed, it would call this.

Reply:
`created_rendering_device` is static variable (3 state), it's already checking it once.